src/usr/local/www/system_camanager.php

```diff
@@ -250,6 +253,12 @@
 
 		$ca['descr'] = $pconfig['descr'];
 
+		if (!isset($pconfig['trusted']) || empty($pconfig['trusted'])) {
+			$ca['trusted'] = false;
+		} else {
+			$ca['trusted'] = $pconfig['trusted'];
+		}
+
```

Comment:
Not sure about this one and subsequent comment that suggests deleting `&& $ca['trusted']`. isset() will return TRUE even if the variable is set FALSE, right? Is it *completely* safe to assume that a boolean FALSE never appears in `global $config`?